map_while_grounded: true

mapping:

  # resolution on the lowest level (fractor = 0, depth = max depth)
  resolution: 0.2 # [m]

  # fractor = how many levels from the lowest level (the highest res)
  #           should we work on
  # current_resolution = resolution * 2 ^ fractor
  initial_fractor: 0

global_map:

  # the publisher rate of the full map
  rate: 1.0 # [Hz]

  # should we prune the map before publishing?
  compress: true

  publish_full: true # should publish map with full probabilities?
  publish_binary: false # should publish map with binary occupancy?

local_map:

  # should create a local map from the global map?
  enabled: true

  # the bounding box distance of the local map
  horizontal_distance: 10.0
  vertical_distance: 10.0

  # the publisher rate of the local map
  rate: 3.0 # [Hz]

  max_computation_duty_cycle: 0.33

  publish_full: true # should publish map with full probabilities?
  publish_binary: true # should publish map with binary occupancy?

unknown_rays:

  update_free_space: true
  clear_occupied: false # beware, setting this to true is very dangerous
  ray_distance: 15.0

sensor_model:
  min_range: 0.4 # [m] ignore points closer than this
  max_range: 10.0 # [m] ignore points further than this
  min: 0.3  # [0, 1] occupancy threshold for definetly free
  max: 0.9  # [0, 1] occupancy threshold for definetly occupied
  hit: 0.9  # [0, 1]
  miss: 0.4 # [0, 1]
